soc/arm/st_stm32/stm32f0/dts_fixup.h

```diff
@@ -188,7 +188,13 @@
 #define DT_TIM_STM32_17_CLOCK_BITS	DT_ST_STM32_TIMERS_40014800_CLOCK_BITS
 #define DT_TIM_STM32_17_CLOCK_BUS	DT_ST_STM32_TIMERS_40014800_CLOCK_BUS
 
-#define DT_WDT_0_NAME                   DT_INST_0_ST_STM32_WATCHDOG_LABEL
+#define DT_WDT_0_NAME       DT_INST_0_ST_STM32_WATCHDOG_LABEL
```

Comment:
Please fix indentation.
Apply everywhere

Reply:
I think it's good now.